### PR TITLE
added html exercise answers

### PR DESCRIPTION
**Pull request by StashaG**
These are my html exercise answers.

### CODE REVIEW

File: week3/1-tuesday/class/exercise2/stacey-answers.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <title>Exercise2</title>
+    </head>
+    <body>
+        
+        <div>
+            <h1>The example is about nesting div tags</h1>
+            <p>This div contains the text and the div tag below.</p>
+        <div>
+            <p>Hi, I'm the div tag that was mentioned in my parent div! I contain this text and two other children divs</p>  
+        </div>
```
Comment (DanStough) [inline week3/1-tuesday/class/exercise2/stacey-answers.html:14]: On the right track here! Just missing the last two nested, third-level `divs`.

File: week3/1-tuesday/class/exercise4/stacey-answers.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<html>
+    <head>
+        <title>Exercise4</title>
+    </head>
+    <body>
+        <h1>Lines. So many lines.</h1>
+        <hr>
+        <br />
+        <hr>
+        <hr>
```
Comment (DanStough) [inline week3/1-tuesday/class/exercise4/stacey-answers.html:11]: You can use the self closing notation with `<hr />` tags as well.

File: week3/1-tuesday/class/exercise5/stacey-answers.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<html>
+    <head>
+        <title>Exercise5</title>
+    </head>
+    <body>
+        <h1>Here's a couple ways you can express yourself</h1>
+        <hr>
+        <ul>
+            <li>If you want to sound normal, just use a regular li tag here</li>
+            <li>If you want to sound interesting, try the italics tag</li>
```
Comment (DanStough) [inline week3/1-tuesday/class/exercise5/stacey-answers.html:11]: Need the italics tag on this line.

**Review by DanStough — APPROVED** [review]
Nice work @StashaG! Take a look at the minor comments below. Going to close the pull request but feel free to fix OYO.

File: week3/1-tuesday/class/exercise5/stacey-answers.html (hunk 3 above)
Comment (StashaG) [reply week3/1-tuesday/class/exercise5/stacey-answers.html:11]: I added italics to line 11. Can you see this change?

**Pull request closed without merging**